README.md

```diff
@@ -92,7 +92,39 @@ There are some shortcuts to get you started:
 
 ## Examples
 
-- Flats list (https://codesandbox.io/s/5wwp76310n)
+### Basics
+
+<table border="0">
+  <tr>
+    <td width="33%" valign="top">
+      <a href="" target="_blank">
+        <img src="" alt="Basic composition image" />
+      </a>
+      <a href="https://codesandbox.io/s/simple-composition-5mvlr" target="_blank">
+        <h4>Basic composition</h4>
+      </a>
+      <p>Combine two UI elements into a single one using Composition.
+    </td>
+    <td width="33%" valign="top">
+      <a href="https://codesandbox.io/s/responsive-props-8m14f" target="_blank">
```

Comment:
Since there are two identical links following each other, maybe let's try wrapping image and heading in a single link?

```diff
- <a>
-   <img>
- </a>
- <a>
-  <h4>
- </a>
+ <a>
+  <img>
+  <h4>
+ </a>
```

Give it a try and let me know.